### DISCUSSION TITLE
Total Supply & Currency Symbol

### DISCUSSION THREAD

**Discussion by mihirmore25** [0xPolygon/polygon-edge#646] [Q&A]
Hi There!

I do have 2 questions as follows: 

1) How to set total supply on local setup of polygon edge ?
&
2) How to add currency symbol of our own like "ABC", "XYZ", etc on polygon edge local setup ?

**Comment by sachinjain551836** [comment]
I am working in a project, which is used the polygon edge as base technology.
I need to create a new blockchain & currency.
So some application we will build over this blockchain & this newly created currency work as GAS for all transaction.

**Comment by laviniat1996** [comment]
Hi @mihirmore25 , 
Thank you for your questions!

1. You can set a premined balance to accounts (addreses) by using the `--premine` flag.  Please note that this is a configuration from the genesis file, and it cannot be changed later. 
Example on how to use the `--premine` flag:` --premine=0x3956E90e632AEbBF34DEB49b71c28A83Bc029862:1000000000000000000000`
This sets a premined balance of 1000eth to 0x3956E90e632AEbBF34DEB49b71c28A83Bc029862 (the amount from the argument is in gwei). Find more about it [here](https://docs.polygon.technology/docs/edge/get-started/set-up-ibft-locally) .

2. For the token symbol, you cannot configure it from the chain setup. But, you can change it when you add the network to a wallet, like Metamask, for example.

**Reply by mihirmore25** [reply]
For the token symbol we do have to deploy our local blockchain on production level so that other people(or EOA's) can send transactions in our currency @laviniat1996 can we change token symbol for that other than adding network to a wallet, like Metamask.

**Reply by laviniat1996** [reply]
@mihirmore25 The token symbol is just a UI thing, so it cannot be configured or harcoded anywhere in the network.

**Reply by sachinjain551836** [reply]
Hey Lavinia, 
We want to use our coin symbol & which can be utilized for burning fees as well as gas consumption whenever any transaction happens. It would be great if any support provides..  

Our ultimate goal to make our Blockchain and currency/token using Polygon Edge framework and later on we build various application upon this.. But ultimate transaction would in our native currency/token/symbol.

Regards,
Sachin

**Reply by laviniat1996** [reply]
Hi @sachinjain551836 ,

For now, that's the only way of setting up the coin symbol. Regarding burning fees, Edge doesn't have that option at the moment. If you want to implement it yourselves, you would have to make a fork from our Edge repo.

**Reply by sachinjain551836** [reply]
Thank You Lavina for response.
